### DISCUSSION TITLE
The local file always does not exist at startup

### DISCUSSION THREAD

**Discussion by Daoting** [unoplatform/uno#8090] [Q&A] [answered]
I need to check for the existence of some local files in `Application.OnLaunched`. But these files do not exist after each restart. I checked that these files do exist in IndexedDB's /local. If I add a delay then everything works fine. I don't know what to do.

```
      protected override async void OnLaunched(LaunchActivatedEventArgs p_args)
      {
         await Task.Delay(100);
          await Startup.Launch(Stub, p_args.Arguments);
      }
```

**Comment by jeromelaban** [comment, marked as answer]
Local files are "restored" by IDBFS automatically some time after the application startup. You need to await any method from the `StorageFolder` class first to access the files. See [this page for more details](https://platform.uno/docs/articles/features/windows-storage.html?q=storage#webassembly-support).

**Reply by Daoting** [reply]
Thanks.
